### PR TITLE
fix sidenav initials issue, delete dr_bak folder in pages

### PR DESCRIPTION
**Pull request by toyaaponesto**
-fix SideNav button initials not showing in first page load
-delete dr_bak folder

### CODE REVIEW

File: apps/portal/src/pages/[id]/applicant-endorsement/[vpp_id].tsx [hunk 1]
```diff
@@ -21,13 +24,30 @@ export default function AppEndSummary({
 }: AppEndSummaryProps) {
   const publication = useAppEndStore((state) => state.selectedPublication);
   // const selectedApplicants = useAppEndStore((state) => state.selectedApplicants);
+  const [navDetails, setNavDetails] = useState<NavButtonDetails>();
 
+  useEffect(() => {
```
Comment (dev-gscwd) [inline apps/portal/src/pages/[id]/applicant-endorsement/[vpp_id].tsx:29]: Remove this empty useEffect function

Comment (toyaaponesto) [reply apps/portal/src/pages/[id]/applicant-endorsement/[vpp_id].tsx]: Removed unused useEffect

**Review by dev-gscwd — COMMENTED** [review]
Please see comments

**Review by toyaaponesto — COMMENTED** [review]
removed useEffect in [vpp_id].tsx

**Review by dev-gscwd — APPROVED** [review]

**Pull request merged**